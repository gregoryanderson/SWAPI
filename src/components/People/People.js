import React from 'react';
import './People.css';

function People () {
    return (
        <p>hey alyssa</p>
    )
}

export default People;