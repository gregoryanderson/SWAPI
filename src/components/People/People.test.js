import React from 'react';
import { shallow } from 'enzyme';
import People from './People'

describe('People', () => {
    it('should look like the snapshot', () => {
        const wrapper = <People />;
        expect(wrapper).toMatchSnapshot();
    })
})