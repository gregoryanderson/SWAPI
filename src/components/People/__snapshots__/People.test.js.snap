// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`People should look like the snapshot 1`] = `<People />`;
